src/index.html

```diff
@@ -8,11 +8,24 @@
 </head>
 <body>
     <div id="controls" class="hidden">
-      <div id="file-selector">
+      <div id="options-panel">
+        <div>
+          <h3>Options</h3>
+          <label>Fit:
+            <select name="fitTypes" id="fitType-selected">
+              <option value="Default" selected="yes">Default</option>
```

Comment:
:information_source:  rename if type is renamed

Reply:
It's done in the same commit of renaming 😉